src/controllers/delivery.ts

```diff
@@ -84,7 +85,11 @@ export class DeliveryController {
       );
     }
     ctx.body = { output: deliveryResponse };
-    ControllerUtility.deliveryPostProcess(ctx, deliveryResponse.status);
+    if (isDefinedAndNotNullAndNotEmpty(deliveryResponse.authErrorCategory)) {
```

Comment:
Only in case of oauth the non-200 status present in the payload will be sent as http status code from transformer to server